scripts/example_dataset_splits.py

```diff
@@ -0,0 +1,47 @@
+import argparse
+
+from paint import PAINT_ROOT
+from paint.data.dataset_splits import DatasetSplitter
+from paint.util import set_logger_config
+
+set_logger_config()
+
+
+if __name__ == "__main__":
+    # Read in arguments.
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--input_file",
+        type=str,
```

Comment:
Should that also be a path-like object?

Reply:
As far as I know "path-like" object cannot be given as inputs to a command line - they can only be adjusted once read in....